torch_glow/setup.py

```diff
@@ -124,6 +125,7 @@ def _run_cmake(self):
                 # PyTorch cmake args
                 '-DPYTORCH_DIR={}'.format(
                     os.path.dirname(os.path.realpath(torch.__file__))),
+                '-DTORCH_GLOW={}'.format(FILE_DIR),
```

Comment:
What is this?

Reply:
Normally we should use include paths relative to "glow" folder, otherwise we would need (as in this case) path to the include file from different folder. Because I have moved test file into a separate folder, test makefile needs include path to other torch_glow files.